CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
 - Resource bundle fallback message [#1233](https://github.com/epam/sap-commerce-intellij-idea-plugin/pull/1233)
 - AnAction text in toolbar configuration [#1236](https://github.com/epam/sap-commerce-intellij-idea-plugin/pull/1236)
 
+### Fixes
```

Comment:
ups, you're one day after we've release 2024.3.0, can you please move it to `2024.3.1` ?

Reply:
yes

Reply:
@mlytvyn  is there still something to do here from my site?
Could you create a release Branch `release/2024.3.1` so that i can set the target correctly?

Reply:
let's merge to `main`, add new section in the `CHANGELOG.log` file for `2024.3.1`

Reply:
updated CHANGELOG.md